lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -101,7 +91,11 @@
                 [type]=property.inputType
                 class="adf-property-value"
                 title="{{property.label | translate }}"
-                [ngClass]="{ 'adf-textitem-clickable-value': !isEditable }"
+                [ngClass]="{
+                    'adf-property-value-editable': editable,
+                    'adf-textitem-clickable-value': !isEditable,
+                    'adf-property-value-has-error': isEditable && hasErrors
```

Comment:
I don't see this class in scss file for this component. Is not that missing?

Reply:
reusable parent class